### PR TITLE
[LICCPA] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Improvements based on recent Romanian translation document revision 06e89b2. 

This files was not translated in Romanian language in any Win version (neither XP/2k3+, nor in the previous ones). I tried to translate this file as close as possible in the way I did in the previous PRs with this attendum.

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian for review

**Comment by binarymaster** [comment]
@AndreiMiloiu please use "Improvements based on recent Romanian translation document revision 06e89b2e81cd7de029fac2c595a6371eba13d797." instead of "Attendum to https://github.com/reactos/reactos/pull/6517", because that referenced PR is already merged, and the word "addendum" is not very applicable to such situation.

And "Attendum" is not a correct English word.

**Comment by AndreiMiloiu** [comment]
> @AndreiMiloiu please use "Improvements based on recent Romanian translation document revision [06e89b2](https://github.com/reactos/reactos/commit/06e89b2e81cd7de029fac2c595a6371eba13d797)." instead of "Attendum to #6517", because that referenced PR is already merged, and the word "addendum" is not very applicable to such situation.
> 
> And "Attendum" is not a correct English word.

I understand. I will change to the other PRs, too.

File: dll/cpl/liccpa/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,33 +1,35 @@
 /*
- * FILE:       dll/cpl/liccpa/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * TRANSLATOR: Ștefan Fulea (stefan dot fulea at mail dot com)
+ * PROJECT:     ReactOS Exploitation System
```
Comment (binarymaster) [inline dll/cpl/liccpa/lang/ro-RO.rc:2]: The... what? Where do you even get such naming? Maybe I overlooked it, but @GeoB99 🤨

Comment (HBelusca) [reply dll/cpl/liccpa/lang/ro-RO.rc:2]: lmao

Comment (AndreiMiloiu) [reply dll/cpl/liccpa/lang/ro-RO.rc:2]: Sorry, I made a mistake. What is the correct name? I can correct it.

Comment (AndreiMiloiu) [reply dll/cpl/liccpa/lang/ro-RO.rc:2]: This is what I found on the internet.

Comment (HBelusca) [reply dll/cpl/liccpa/lang/ro-RO.rc:2]: First rule is to search in our source code, and also whether there are other suitable instances in the module of interest (liccpa): https://git.reactos.org/?p=reactos.git;a=blob;f=dll/cpl/liccpa/liccpa.c;h=50b4fbb4266741e510e0839d0c6c342cea964b49;hb=e8cf32cf15b25cc0d653ee898768e14dd7ba0d36

**Pull request merged**